web/template/home.gohtml

```diff
@@ -290,8 +292,266 @@
                 </article>
             </article>
         </template>
-        <template x-if="view === home.Views.Main">
-            <article id="main-view" class="relative min-h-full pb-8">
+        <template x-if="state.isCourse()">
+            <article x-data="home.courseContext(state.slug, state.year, state.term)"
+                     x-init="$watch('state', (s) => reload(s.slug, s.year, s.term))"
+                     class="tum-live-course-view">
+                <header>
+                    <div>
+                        <p x-text="course.Slug" class="slug"></p>
```

Comment:
I don't think we need the slug here, maybe the semester would be more useful information

Reply:
Agree.